Sources/HaCWebsiteLib/Utils/DateUtils.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+enum DateUtils {
+  private static let locale = Locale(identifier: "en_GB")
+
+  static let individualDayFormatter: DateFormatter = {
+    let formatter = DateFormatter()
+    formatter.dateStyle = .medium
+    formatter.timeStyle = .none
+    formatter.locale = locale
+
+    return formatter
+  }()
+}
+
+extension Date {
+  static func from(year: Int, month: Int, day: Int, hour: Int, minute: Int, timezone: TimeZone? = nil) -> Date! {
+    let formatter = DateFormatter()
+    formatter.dateFormat = "yyyy-MM-dd HH:mm"
+    formatter.timeZone = timezone ?? TimeZone(identifier: "Europe/London")
+    return formatter.date(from: "\(year)-\(month)-\(day) \(hour):\(minute)")
```

Comment:
I'm flabbergasted that we have to work this hard to provide a _lower_ level representation of a date than what the std lib gives us but it's abstracted away!

People may claim that datestrings are easier to work with in this scenario but I really like the type safety this is giving us.

Reply:
Yeah I think people are too inconsistent with date strings. Will probably end up with months and days mixed up at some point. This constructor seems readable enough to me